main/main.c

```diff
@@ -611,6 +611,7 @@ PHP_INI_BEGIN()
 #ifdef PHP_WIN32
 	STD_PHP_INI_BOOLEAN("windows.show_crt_warning",		"0",		PHP_INI_ALL,		OnUpdateBool,			windows_show_crt_warning,			php_core_globals,	core_globals)
 #endif
+	STD_PHP_INI_ENTRY("syslog.ascii_only",			"0",		 PHP_INI_SYSTEM,	OnUpdateBool,		syslog_ascii_only,		php_core_globals, 		core_globals)
```

Comment:
Well my point is that we introduce `syslog.ascii_only` and then to check utf-8, we will need to add `syslog.utf8_only` or something like that. That's not really flexible IMHO. I would prefer a single ini for that where you can specify required encoding.